pasta_eln/guiStyle.py

```diff
@@ -47,14 +47,15 @@ def getColor(backend:Backend, color:str) -> str:
 
 class TextButton(QPushButton):
   """ Button that has only text"""
-  def __init__(self, label:str, function:Optional[Callable[[],None]], layout:Optional[QLayout], name:str='',
-               tooltip:str='', checkable:bool=False, style:str='', hide:bool=False, backend:Optional[Backend]=None):
+  def __init__(self, label:str, widget:QWidget, identifier:list[Any]=[],
+               layout:Optional[QLayout]=None, tooltip:str='', checkable:bool=False, style:str='',
+               hide:bool=False, iconName:str=''):
     """
     Args:
       label (str): label printed on button
-      function (function): function to be called upon button-click-event
+      widget (QWidget): widget / dialog that host the button and that has the execute function
+      identifier (enum): identifier that is used in called-function: possibly a
```

Comment:
Could you complete the comment here? What identifier is it passed by the calling function?

Reply:
That is the main issue: and I don't know how to explain it. The button uses an "Identifier', such that we called function knows, which button called it. How would you explain it.

Reply:
We shall discuss and see this point, I am not sure, if I understood this rightly!